website/frontend/templates/docs/tags.html

```diff
@@ -460,9 +460,9 @@ <h1 id="advanced_tags">Advanced Tags</h1>
         </tr>
         <tr>
           <td>arranger</td>
-          <td> <a href="https://musicbrainz.org/doc/index.php?title=Arranger_Relationship_Type&amp;action=edit&amp;redlink=1"
+          <td> <a href="#"
```

Comment:
There is some documentation for _arranger_ relationships:
- for a _recording_: https://musicbrainz.org/relationship/22661fb8-cdb7-4f67-8385-b2a8be6c9f0d
- for a _release_: https://musicbrainz.org/relationship/34d5334e-a4c8-4b65-a5f8-bbcc9c81d13d
- for a _work_: https://musicbrainz.org/relationship/d3fd781c-5894-47e2-8c12-86cc0e2c8d08

I’m not sure they are all relevant in the context of Picard tagger, please check that before linking to it.

Reply:
Yes these are relevant, I'll fix this and make a commit.